### DISCUSSION TITLE
video-downloaderhelper for FreeBSD...

### DISCUSSION THREAD

**Discussion by Marietto2008** [aclap-dev/video-downloadhelper#345] [Help and Q&A]
### Your question

Hello devs,

I'm a FreeBSD user. But at the same time I really love your extension,because is the best all around. It works better than any other. Unfortunately your extension installed on Firefox installed on FreeBSD is not able to detect the video. And as I see,the external application is not ready for FreeBSD. Do you plan to extend the compatibility also for FreeBSD ? I really miss it. Actually I use it because I've installed it within the FreeBSD emulation layer,but it's not the same as to use it natively. I feel that's not too hard. So,please,think at this. Thanks.

### Addon & CoApp version and details

a

**Comment by malvarco** [comment]
On FreeBSD, you can install yt-dlp package. Very simple to use on terminal, just type:

` yt-dlp https://very_important.videos_for_me.net/etc_etc`

It supports thousands of sites and has millions of killer options. [https://github.com/yt-dlp/yt-dlp](url)

**Comment by Marietto2008** [comment]
Is there a GUI for yt-dlp ? thanks.

On Fri, Nov 29, 2024 at 5:57 AM malvarco ***@***.***> wrote:

> On FreeBSD, you can install yt-dlp package. Very simple to use on
> terminal, just type:
>
> yt-dlp https://very_important.videos_for_me.net/etc_etc
>
> It supports thousands of sites and has millions of killer options.
> https://github.com/yt-dlp/yt-dlp <http://url>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aclap-dev/video-downloadhelper/discussions/345#discussioncomment-11412185>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFYNC4P3IIQN77UMCHX36T2C7XZ5AVCNFSM6AAAAABSWKHVXOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBRGIYTQNI>
> .
> You are receiving this because you authored the thread.Message ID:
> <aclap-dev/video-downloadhelper/repo-discussions/345/comments/11412185@
> github.com>
>


-- 
Mario.

**Reply by malvarco** [reply]
I don't think so. You should request that on the yt-dlp repository.

**Comment by Marietto2008** [comment]
I don't want yt-dlp. I like video DownloadHelper. I want that it works natively on Firefox in FreeBSD. 
I must say that the new version sometime is able to detect videos. Its better than the old version. Unfortunately sometime it is not able.
